src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -502,6 +502,57 @@ public void getProjectsConciseFilterByTagTest() {
                 """);
     }
 
+    @Test
+    public void getProjectsConciseFilterByTeamTest() {
+        qm.createConfigProperty(
+            ACCESS_MANAGEMENT_ACL_ENABLED.getGroupName(),
+            ACCESS_MANAGEMENT_ACL_ENABLED.getPropertyName(),
+            "true",
+            ACCESS_MANAGEMENT_ACL_ENABLED.getPropertyType(),
+            ACCESS_MANAGEMENT_ACL_ENABLED.getDescription()
+    );
+        // Create project and give access to current principal's team.
+        final var projectB= new Project();
+        projectB.setName("acme-app-b");
+        qm.persist(projectB);
+
+        projectB.addAccessTeam(team);
+
+        // Should not return results for partial matches.
+        Response response = jersey.target(V1_PROJECT + "/concise")
+                .queryParam("team", "f")
+                .request()
+                .header(X_API_KEY, apiKey)
+                .get();
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getHeaderString(TOTAL_COUNT_HEADER)).isEqualTo("0");
+        assertThat(getPlainTextBody(response)).isEqualTo("[]");
+
+        // Should return results for exact matches.
+        response = jersey.target(V1_PROJECT + "/concise")
+                .queryParam("team", team.getName())
+                .request()
+                .header(X_API_KEY, apiKey)
+                .get();
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getHeaderString(TOTAL_COUNT_HEADER)).isEqualTo("1");
+        assertThatJson(getPlainTextBody(response)).isEqualTo("""
+                [
+                  {
+                    "uuid": "${json-unit.any-string}",
+                    "name": "acme-app-b",
+                    "active": true,
+                    "taeam": [
```

Comment:
Looks like a typo sneaked in here :detective: 

```suggestion
                    "teams": [
```

Reply:
Thanks for that catch!